build-system/tasks/presubmit-checks.js

```diff
@@ -923,9 +933,9 @@ function matchTerms(file, terms) {
     const {whitelist, checkInTestFolder} = terms[term];
     // NOTE: we could do a glob test instead of exact check in the future
     // if needed but that might be too permissive.
-    if (Array.isArray(whitelist) &&
+    if (isInBuildSystemFixtureFolder(relative) || (Array.isArray(whitelist) &&
```

Comment:
@rsimha do you mind re-reviewing this part. had to whitelist files in build-system/babel-plugins test/fixtures

Reply:
Done. Couple comments.